rnd/autogpt_server/autogpt_server/blocks/search.py

```diff
@@ -135,7 +135,7 @@ class GetOpenWeatherMapBlock(Block, GetRequest):
     class Input(BlockSchema):
         location: str
         api_key: BlockSecret = SecretField(key="openweathermap_api_key")
-        use_celsius: bool
+        use_celsius: bool = True
```

Comment:
I'm sorry for being anti-US here 🦅 but we had to choose a value.

Reply:
![Screenshot 2024-08-23 at 4 33 00 PM](https://github.com/user-attachments/assets/161dd1eb-43a9-40ee-a0ed-80cc239ee28c)
i think this was the right choice 😅

Reply:
Heartbreaking